### PR TITLE
Add cypress wrapper

### PR DESCRIPTION
**Pull request by janineahn**
Regarding #8, this adds a helper script for rerouting cypress via your hosts file.

### CODE REVIEW

**Comment by agoldis** [comment]
Thanks a lot @janineahn  and @redaxmedia! 

---

I am planning to have this tool in a separate repo. I will approve and merge the PR to persist your contribution, and then will extract it to a separate repo and release to npm. Are you ok with that?

**Comment by redaxmedia** [comment]
Haven the CLI inside another project totally makes sense cause using `npx sorry-cypress` would fetch to much unneeded files and dependencies.

Our solution comes with some drawbacks that should be considered. It worked for us but needs rework to work for everyone properly!?

1. Only works with OS that has `etc/hosts`
1. Only works with HTTPS on the target machine*
1. Only works with port 443 on target machine
1. Has fixed arguments for cypress run

*Could properly fixed via HTTPS inside Docker but that's seriously a big task

File: bin/sorry-cypress.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+const cypressArray = [
+    'run',
+    '--record',
+    '--parallel',
+    '--key=' + process.env.SORRY_CYPRESS_RECORD_KEY,
+    '--ci-build-id=' + process.env.SORRY_CYPRESS_BUILD_ID
+];
+
+// exit on windows
+if (process.platform === 'win32') {
```
Comment (redaxmedia) [inline bin/sorry-cypress.js:22]: Exit on Windows can be removed as hostile supports it too

**Comment by redaxmedia** [comment]
@agoldis You are welcome

**Pull request merged**